PubSub/src/Topic.php

```diff
@@ -94,14 +110,20 @@ class Topic
      * @param string $name The topic name
      * @param bool $encode Whether messages should be base64 encoded.
      * @param array $info {
-     *     A [Topic](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics
-     *
-     *     @type string name The name of the topic.
-     *     @type array $labels Key value pairs used to organize your resources.
-     *     @type string $kmsKeyName The resource name of the Cloud KMS
-     *           CryptoKey to be used to protect access to messages published on this
-     *           topic. The expected format is
-     *           `projects/my-project/locations/kr-location/keyRings/my-kr/cryptoKeys/my-key`.
+     *        A [Topic](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics
+     *
+     *        @type string name The name of the topic.
+     *        @type array $labels Key value pairs used to organize your resources.
+     *        @type string $kmsKeyName The resource name of the Cloud KMS
+     *              CryptoKey to be used to protect access to messages published on this
+     *              topic. The expected format is
+     *              `projects/my-project/locations/kr-location/keyRings/my-kr/cryptoKeys/my-key`.
+     *        @type bool $enableCompression Flag to enable compression subject
+     *              tosize of the message. Set the flag to `true` to enable
+     *              compression. Defaults to `false`.
+     *        @type int $compressionBytesThreshold The threshold byte size
+     *              above which messages are compressed. This only takes effect
+     *              if `enableCompression` is set to `true`. Defaults to `240`.
```

Comment:
Default isn't set here, infact it throws if not set and compression is enabled.
```suggestion
     *              if `enableCompression` is set to `true`.
```